public/index.html

```diff
@@ -56,11 +56,17 @@ <h3>Queue batches of lightning channels to open in a single transaction</h3>
 							<button type="button" class="is-circle colored-button" onclick="remove_channel()">-</button>
 						</x-flex>
 					</p>
+					<div>
 				</fieldset>
-				<div>
-					<label for="fee_rate">Maximum fee rate (sats/vB)</label>
```

Comment:
Accidentally removed?

Reply:
Weird this went away when i rebased locally, maybe github being confused wth the merge

Reply:
Added back in. Thanks for catching this